src/diffusers/models/unet_2d.py

```diff
@@ -1,14 +1,27 @@
-from typing import Dict, Optional, Tuple, Union
+from dataclasses import dataclass
+from typing import Optional, Tuple, Union
 
 import torch
 import torch.nn as nn
 
 from ..configuration_utils import ConfigMixin, register_to_config
 from ..modeling_utils import ModelMixin
+from ..utils import ModelOutput
 from .embeddings import GaussianFourierProjection, TimestepEmbedding, Timesteps
 from .unet_blocks import UNetMidBlock2D, get_down_block, get_up_block
 
 
+@dataclass
+class UNet2DOutput(ModelOutput):
+    """
+    Args:
+        sample (`torch.FloatTensor` of shape `(batch_size, num_channels, height, width)`):
+            Hidden states output. Output of last layer of model.
+    """
+
+    sample: torch.FloatTensor = None
```

Comment:
General comment for all the `ModelOutput` subclasses: do we need to initialize values to `None`? I'd rather remove all the `None`s as we never really instantiate the outputs with empty values. I think it's clearer too, since it avoids user confusion about how to deal with potential `None` values.

Reply:
Yes, that's true, but then I'd use `None` for those other outputs that really are optional. It doesn't look like any of the current ones could be skipped.

Reply:
Hmm yeah fair point @pcuenca ! Adapting this